src/components/register-in-contest/register-screenshots.jsx

```diff
@@ -0,0 +1,71 @@
+"use strict";
+
+import "blueimp-file-upload";
+import "../../../node_modules/blueimp-file-upload/css/jquery.fileupload.css";
```

Comment:
Ma gandesc ca pe server poti sa ai `node_modules` si in alta parte si pe urma o sa stai sa faci debug ampulea ca nu ti se load-uie niste css-uri.

Le lasam asa deocamdata, dar punem issue sa facem ceva cu ele